apps/fluent-tester/src/E2E/common/BasePage.ts

```diff
@@ -1,4 +1,5 @@
 const DUMMY_CHAR = '';
+const COMPONENT_SCROLL_COORDINATES = { x: -0, y: -40 }; // These are the offsets. Y is negative because we want the touch to move up (and thus it scrolls down)
```

Comment:
-0? is down the only way we want to scroll ever?  Can we scroll more directly based on layout information or otherwise?